### PR TITLE
[Debt] Migrate search flows test to playwright

### LINKED ISSUE #10493: ♻️ Move e2e to Playwright - search-workflows.cy.ts
## ♻️ Debt/Refactor

Time to move our Cypress tests to Playwright
apps/e2e/cypress/e2e/talentsearch/search-workflows.cy.ts

## ✅ Acceptance Criteria

- [ ] Test moved to Playwright : search-workflows.cy.ts

## 🛑 Blockers

```[tasklist]
### Blocked By
```

### PR DESCRIPTION
**Pull request by esizer**
🤖 Resolves #10493 

## 👋 Introduction

Migrates the search workflows test from cypress to playwright

## 🧪 Testing

1. Test has similar coverage
2. Test passes

### CODE REVIEW

File: apps/playwright/tests/search-workflows.spec.ts [hunk 1]
```diff
@@ -0,0 +1,226 @@
...
+    await streamFilter.selectOption({ label: "Database Management" });
+
+    await expectNoCandidate(appPage.page);
+
+    await streamFilter.selectOption({
+      label: "Business Line Advisory Services",
+    });
+
+    await expect(poolCard).toBeVisible();
+
```
Comment (petertgiles) [inline apps/playwright/tests/search-workflows.spec.ts:148]: No education requirement test?

File: apps/playwright/tests/search-workflows.spec.ts [hunk 2]
```diff
@@ -0,0 +1,226 @@
...
+    await appPage.page.getByRole("checkbox", { name: /ontario/i }).click();
+
+    await expectNoCandidate(appPage.page);
+
+    await appPage.page.getByRole("checkbox", { name: /atlantic/i }).click();
+
+    await expect(poolCard).toBeVisible();
+
```
Comment (petertgiles) [inline apps/playwright/tests/search-workflows.spec.ts:156]: No working language ability test?

**Review by petertgiles — CHANGES_REQUESTED** [review]
Seems to run OK!  The test seems a little weaker though?

File: apps/playwright/tests/search-workflows.spec.ts (hunk 1 above)
Comment (esizer) [reply apps/playwright/tests/search-workflows.spec.ts:155]: okay, I think I got all of the filters covered now with [add more filters to test coverage](https://github.com/GCTC-NTGC/gc-digital-talent/pull/10560/commits/dfde102b82fac0c2f8d570f7675539768eb91a88)

**Review by petertgiles — APPROVED** [review]

**Pull request merged**